editors/ne/files/patch-makefile.diff

```diff
@@ -1,11 +1,22 @@
---- makefile.orig	2021-05-27 10:41:44.000000000 -0400
-+++ makefile	2021-05-27 10:41:59.000000000 -0400
-@@ -36,7 +36,7 @@
- 	( cd doc; make pdf )
+diff -urN a/makefile b/makefile
+--- a/makefile	2023-11-04 19:35:24.000000000 -0400
++++ b/makefile	2023-11-04 20:02:11.000000000 -0400
+@@ -27,7 +27,8 @@
+ 
+ 
+ build: docs
+-	( cd src; $(MAKE) clean; $(MAKE) NE_GLOBAL_DIR=$(PREFIX)/share/ne; $(STRIP) ne )
++	( cd src; $(MAKE) clean; $(MAKE) NE_GLOBAL_DIR=$(PREFIX)/share/ne )
++	$(STRIP) src/ne
+ 
+ docs:
+ 	( cd doc; $(MAKE) )
+@@ -36,7 +37,7 @@
+ 	( cd doc; $(MAKE) pdf )
  
  version:
 -	./version.pl VERSION=$(VERSION)
 +	perl version.pl VERSION=$(VERSION)
```

Comment:
Maybe reference the perl executable as a make variable so that you can pass in the path to MacPorts Perl via the `build.pre_args`:

```suggestion
+	$(PERL) version.pl VERSION=$(VERSION)
```

Then in the Portfile:
```
depends_build-append \
                    port:perl5 \
                    port:texinfo
...
build.pre_args-append \
                    PERL=${prefix}/bin/perl5
build.target        build
```

Reply:
Will do. Thanks :)